authorizers/iconik/body-authorizer.ts

```diff
@@ -24,7 +25,6 @@ type Payload = CustomActionPayload & { auth_token?: string };
 
 export async function authorizeWithIconikCustomAction(customActionBody: Payload): Promise<IconikContext> {

```

Comment:
I think there should be a comment that says "Prefer authorization by header", but more detailed why.